pybtp/types.py

```diff
@@ -86,6 +86,8 @@ class AdFlags:
     sim_le_br_edr_contr = 0x08
     sim_le_br_edr_host = 0x10
 
+class AdDuration:
+    forever = 4294967295
```

Comment:
Just a nitpick: you can use hexadecimal values in Python, so I think it would be more readable if the value was 0xFFFFFFFF